common/common.go

```diff
@@ -9,23 +9,28 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func Run(framework *clustertest.Framework, cluster *application.Cluster) {
+var (
+	Framework *clustertest.Framework
+	Cluster   *application.Cluster
+)
+
+var _ = Describe("Common tests", func() {
```

Comment:
The downside to this change is that it's less explicit. You have to know that importing the package will set up the tests.
I've left this as a separate commit so we can revert it if we decide we don't want to go that way. Running just `common.Run()` with the now exported `Framework` and `Cluster` vars works

Reply:
I prefer the explicit version, but I'm not blocking this.

Reply:
I think I'm also leaning towards the more explicit version. But I want to hear other people's opinions

Reply:
I also think explicit as ordering might be needed.